sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/MetricsQueryAsyncClient.java

```diff
@@ -80,10 +95,12 @@
      * @return List of metrics namespaces.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
+    @SuppressWarnings("deprecation")
```

Comment:
Has the swagger deprecated these?

Reply:
No, this is not a deprecated method. It is just suppressing warnings since the `mapPage` in azure core is marked as deprecated.